src/cook-web/src/components/dnd-kit-sortable-tree/SortableTree.tsx

```
@@ -1,427 +1,427 @@
+  const [overId, setOverId] = useState<UniqueIdentifier | null>(null);
+  const [offsetLeft, setOffsetLeft] = useState(0);
+  const [currentPosition, setCurrentPosition] = useState<{
+    parentId: UniqueIdentifier | null;
+    overId: UniqueIdentifier;
+  } | null>(null);
+
+  const flattenedItems = useMemo(() => {
+    const flattenedTree = flattenTree(items);
+    const collapsedItems = flattenedTree.reduce<UniqueIdentifier[]>(
+      (acc, { children, collapsed, id }) =>
+        collapsed && children?.length ? [...acc, id] : acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)